pkg/reviveplanner/vcparser/vc_parser.go

```diff
@@ -0,0 +1,109 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vcparser
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/vertica/vertica-kubernetes/pkg/reviveplanner/util"
+)
+
+type Parser struct {
+	Cluster Cluster
```

Comment:
Why isn't there a `Log` like in admintools'?

Reply:
We had no reason to log a message (yet), so I never added it.